airgun/views/oscapcontent.py

```diff
@@ -60,7 +60,7 @@ def is_displayed(self):
 
 
 class SCAPContentEditView(SCAPContentCreateView):
-    scap_file_name = Text("//b")
+    scap_file_name = Text('//div[@class="col-md-4"]/b')
```

Comment:
just sharing knowledge, it is better to use: `//label[contains(., 'Scap File')]/following-sibling::div/b`